pkg/apicast/apicast.go

```diff
@@ -67,9 +71,21 @@ func (a *APIcast) deploymentVolumeMounts() []v1.VolumeMount {
 		})
 	}
 
+	for _, customPolicySpec := range a.options.CustomPolicies {
+		volumeMounts = append(volumeMounts, v1.VolumeMount{
```

Comment:
I'm curious about this behavior.
What would happen if the user updates the secret content? Is that immediately visible in the container's filesystem?
I'm guessing that independently on that apicast might or might not be able to react to file changes once it's running. Do you know about that too?

Reply:
I will do some testing about this. My guess, apicast will not notice the change even if the new content is available in the volume.

If updating the content of the secret does not automatically make it available to the container (which I do not think so):
* Watch for secret updates and redeploy
* Document that secret updates are not visible to the apicast and you need to redeploy after secret has been updated. Or create a new secret and change reference in the CR. 

For the first iteration, I would go for the doc solution. wdyt?

Reply:
seems reasonable to me :+1:

Reply:
doc added